fimsauthz-api/fims-authz.yaml

```diff
@@ -435,3 +480,41 @@ components:
         error_description:
           type: string
           example: "Human readable description of the error."
+    JsonWebKeySet:
+      type: "object"
+      properties:
+        keys:
+          type: "array"
+          items:
+            $ref: "#/components/schemas/JsonWebKey"
+    JsonWebKey:
```

Comment:
@arkits recommend making the known-required fields explicit now.  https://tools.ietf.org/html/rfc7517#section-4. A quick look at the RFC I think shows everything as optional except kty.

Reply:
Is there a reason for all of the non-7517 fields?  Are they related to known algorithms that will be used?  x, y , d, n, e, k... those are all specific algorithm fields.  Why?  Relatedly, why don't we list the known-optional JWK fields for future proofing?

Reply:
Ok.. i see below that kty is required.  please ignore that comment.